<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>331</width>
    <height>306</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>播放列表</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="btnAdd">
           <property name="text">
            <string>添加</string>
           </property>
           <property name="icon">
            <iconset resource="../VideoPlayer/res.qrc">
             <normaloff>:/images/images/316.bmp</normaloff>:/images/images/316.bmp</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnRemove">
           <property name="text">
            <string>移除</string>
           </property>
           <property name="icon">
            <iconset resource="../VideoPlayer/res.qrc">
             <normaloff>:/images/images/318.bmp</normaloff>:/images/images/318.bmp</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnClear">
           <property name="text">
            <string>清空</string>
           </property>
           <property name="icon">
            <iconset resource="../VideoPlayer/res.qrc">
             <normaloff>:/images/images/214.bmp</normaloff>:/images/images/214.bmp</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QListWidget" name="listWidget"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="btnPlay">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../VideoPlayer/res.qrc">
          <normaloff>:/images/images/620.bmp</normaloff>:/images/images/620.bmp</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnPause">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../VideoPlayer/res.qrc">
          <normaloff>:/images/images/622.bmp</normaloff>:/images/images/622.bmp</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnStop">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../VideoPlayer/res.qrc">
          <normaloff>:/images/images/624.bmp</normaloff>:/images/images/624.bmp</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnPrevious">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/images/images/616.bmp</normaloff>:/images/images/616.bmp</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnNext">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/images/images/630.bmp</normaloff>:/images/images/630.bmp</iconset>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>68</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnSound">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../VideoPlayer/res.qrc">
          <normaloff>:/images/images/volumn.bmp</normaloff>:/images/images/volumn.bmp</iconset>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="sliderVolumn">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="LabCurMedia">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>无曲目</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="sliderPosition">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="LabRatio">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>进度</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
